### DISCUSSION TITLE
fastlane-2.158.0 - firebase_app_distribution step - iOS

### DISCUSSION THREAD

**Discussion by amitcomo** [fastlane/fastlane#17173] [Q&A]
Hi,

I installed fastlane version -2.158.0.
I execute it in the following way:
`    firebase_app_distribution(
              app: "__firebaseAppId__",
              testers: "__testers__",
              firebase_cli_path: "path_to_firebase_cli",
              ipa_path: "__ipaPath__",
              firebase_cli_token: "__token__",
              release_notes_file: "../version.txt"
          )`

The output of this call:

> 
> Authenticatin with --firebase_cli_token parameter
> (Lane Context...)
> **App Distribution could not generate credentials from the refresh token specified**

Any idea what was changed in this version that causing this error?

**Comment by jongwonatwork** [comment]
Hi, @amitcomo 
I have the same problem. 
Have you solved the problem?
